03/components/Category.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+function Category({children}) {
+
+    return (
+        <>
+            <h2>Category</h2>
+            {children}
+        </>
+    );
+}
```

Comment:
Ten komponent jest identyczny jak `Cart` - może warto stworzyć jeden np. `Group` i go wykorzystać w obu miejscach :)

Reply:
Oo, dzięki! Zwróciłem na to uwagę i właśnie zastanawiałem się, czy nie robić czegoś z tym, żeby zachować DRY :).